README.md

```diff
@@ -1,3 +1,8 @@
+# Disclaimer
+ `main` branch is stable but currently under development so documentation may not be fully up to date with the current feature set.
+
+It is advised that if one is to look at documentation, they should refer a specific tag/release.
```

Comment:
I would change the word on this a bit.

```
Please refer to documentation for the specific your specific release.
```

Maybe not that verbatim but I think the wording `if one is to look at` you have now sounds a bit odd.

Reply:
I agree with Corbin. I would go with this:

> While the `main` branch is stable, it is still under development and documentation may not fully reflect the current feature set. Please refer to documentation for your specific release.